odrive_state_controller/src/odrive_state_controller.cpp

```diff
@@ -102,6 +117,24 @@ bool OdriveStateController::clearErrorsCallback(std_srvs::Empty::Request& /* req
   return 1U;
 }
 
+bool OdriveStateController::setAxisStateCallback(odrive_state_msgs::SetAxisState::Request& request,
+                                                 odrive_state_msgs::SetAxisState::Response& /* response */) {
+  odrive_axis_state_cmd_[request.joint_name].store(request.axis_state);
```

Comment:
Not worth changing, but might be nice to a) accept a list of joints and b) verify the requested state is a valid state. For the next dog :P

Reply:
done